test/parallel/test-http-set-timeout-server.js

```diff
@@ -213,3 +213,16 @@ test(function fastTimeout(cb) {
     });
   }));
 });
+
+test(function serverTimeout(cb) {
+  const server = http.createServer();
+  server.listen(common.mustCall(() => {
+    const timeoutValue = 50;
+    const s = server.setTimeout(timeoutValue);
+    assert.equal(s.timeout, timeoutValue);
```

Comment:
Use `assert.strictEqual()` instead?

Reply:
Thanks @Trott for quick review, I'll post an update in new commit